src/Multisig.sol

```diff
@@ -225,6 +225,8 @@ contract Multisig is IMultisig, IMembership, PluginUUPSUpgradeable, ProposalUpgr
         // Create the proposal
         Proposal storage proposal_ = proposals[proposalId];
         proposal_.metadataURI = _metadataURI;
+
+        // Q(danielw): shall we validate `_destinationPlugin`?
```

Comment:
Permission checks are made on the destination plugin, rather than at the source. 

The `createProposal()` method on `OptimisticTokenVotingPlugin` has a modifier `auth(PROPOSER_PERMISSION_ID)`, which is only granted to the two multisig's. 

Regarding the case where `_destinationPlugin` was a random address, there wouldn't be an inventive to do so because the multisig's have no permission anywhere else.